examples/simple-example/package.json

```diff
@@ -15,18 +15,19 @@
     "vite-plugin-solid": "^2.9.1"
   },
   "dependencies": {
+    "@floating-ui/dom": "^1.6.0",
     "@tailwindcss/typography": "^0.5.10",
-    "@tiptap/core": "^2.1.16",
-    "@tiptap/extension-bubble-menu": "^2.1.16",
-    "@tiptap/pm": "^2.1.16",
-    "@tiptap/starter-kit": "^2.1.16",
+    "@tiptap/core": "^3.0.1",
+    "@tiptap/extension-bubble-menu": "^3.0.1",
+    "@tiptap/pm": "^3.0.1",
+    "@tiptap/starter-kit": "^3.0.1",
     "solid-js": "^1.8.12",
-    "solid-tiptap": "0.7.0",
+    "solid-tiptap": "workspace:*",
```

Comment:
Months ago, for some reason, Lerna had issues with using PNPM as the the configurable package manager, so for now I use NPM for publishing with Lerna. You probably want to revert this line change for now.

In the future, I might have to move all my projects to changesets.

Reply:
Changed this. Thanks for your review 😄